spec/requests/api/v1/endgame_spec.rb

```diff
@@ -70,12 +70,20 @@
   end
 
   describe "sad path", :vcr do
-  
     it "evaluates the parms to confirm they are integers" do
       wins   = "d"
```

Comment:
I like the invalid param being completely invalid, should we allow / tests for more different types of sad paths?